compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -458,7 +462,8 @@ impl CodegenBackend for GotocCodegenBackend {
                         );
                     }
                     MonoItem::GlobalAsm(_) => {
-                        warn!(
+                        rmc_warn!(
+                            WarningType::Other,
```

Comment:
Lets not leave this as Other.  We should have a good name that can be searched for. Maybe even just `GlobalAsm`

Reply:
Done.